tools/internal_ci/linux/grpc_xds_k8s_xlang_java.cfg

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Config file for the internal CI (in protobuf text format)
+
+# Location of the continuous shell script in repository.
+build_file: "grpc/tools/internal_ci/linux/grpc_xds_k8s_xlang_java.sh"
```

Comment:
Is it possible to pass arguments to the script? So I can have just 1 .sh file but multiple .cfg files passing different arguments (or is env var the way to go?)

Reply:
I don't think it's possible to pass args to the script. But you can still have single `.sh` and multiple cfg's by using `env_vars` field. Or even make it a hardcoded array in the script itself.

Reply:
used hard coded array